Utilities/WDLs/DownsampleAndCollectCoverage.wdl

```diff
@@ -181,6 +181,11 @@ task Downsample {
             -CREATE_INDEX false \
             -REFERENCE_SEQUENCE ~{ref_fasta}
         
+        # Create reference cache for indexing CRAM file
+        seq_cache_populate.pl -root ./ref/cache ~{ref_fasta}
```

Comment:
Wait where does this script come from? Is it already in the docker?

Reply:
@rickymagner Yep, `apt install samtools` installs it into `/usr/bin`